src/components/AnalyticsDrawer/AnalyticsDrawer.tsx

```diff
@@ -1,30 +1,52 @@
+import CircularProgress from 'material-ui/CircularProgress';
 import { Drawer } from 'material-ui'
```

Comment:
You can add this to the existing import. A bonus is that naming is enforced and can be more easily statically analyzed, because it's a named export:

```diff
- import CircularProgress from 'material-ui/CircularProgress'
import { CircularProgress, Drawer } from 'material-ui'
```

Also, the `;` should not pass lint (we should update the build to also run lint as part of it). If you use your editor's autoformat feature ([VSCode](https://code.visualstudio.com/docs/languages/typescript), [Sublime](https://github.com/Microsoft/TypeScript-Sublime-Plugin), [Atom](https://github.com/TypeStrong/atom-typescript) et all. support it) you won't have to think about this.